about.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title></title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="./styles/style.css">
+        <link rel="stylesheet" href="./styles/about_style.css">
+        <!-- For social media icons: -->
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    </head>
+    <body>
+        <header>
+            <h1 class="my-name"><a href="./index.html">WHIT SUNDBY</a></h1>
+            <nav class="navbar">
+                <a class="current-page" href="./about.html">About</a></li>
+                <a href="./portfolio.html">Portfolio</a></li>
+                <a href="./blog.html">Essays</a></li>
```

Comment:
You have close `li` tags, but no `ul` and no open `li` tags.  You should have an `ul` or something as well as `li` start tags.